app/view/twig/changelog/_blocks.twig

```diff
@@ -188,10 +188,12 @@
                             </td>
                         {% else %}
                             <td class="change-log-item">
-                                {{ field.before.render }}
+                                {# {{ field.before.render }} #}
+                                {{ dump(field.before.render)}}
                             </td>
                             <td class="change-log-item">
-                                {{ field.after.render }}
+                                {# {{ field.after.render }} #}
+                                {{ dump(field.after.render)}}
```

Comment:
Erm … you sure about that?

Reply:
Nope, that's part of the WIP. It's there so it won't break on Repeaters/Blocks. I need to work on that.